src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+body {
+  margin-right: 8px;
+  margin-left: 8px;
```

Comment:
You are applying left and right margins separately. It's better to use shorthand property to keep the code clean and easy to read. For example, you could write `margin: 0 8px;` instead of two separate lines.